backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BPPrintFormat.java

```diff
@@ -37,40 +42,46 @@
  *
  */
 
+@Getter
 @Value
 @Builder
 public class BPPrintFormat
 {
-	@Getter
-	private final BPartnerId bpartnerId;
+	@NonNull
+	BPartnerId bpartnerId;
+
+	@Nullable
+	DocTypeId docTypeId;
 
-	@Getter
-	private final int docTypeId;
+	@Nullable
+	AdTableId adTableId;
 
-	@Getter
-	private final int adTableId;
+	int printFormatId;
 
-	@Getter
-	private final int printFormatId;
+	int bpPrintFormatId;
 
-	@Getter
-	private final int bpPrintFormatId;
+	@Nullable
+	BPartnerLocationId bPartnerLocationId;
+
+	@Nullable
+	PrintCopies printCopies;
```

Comment:
why this is nullable? can't be default 1.

Reply:
Was nullable at the beginning and forget to adjust after i changed it. Fixed